pkg/scheduler/framework/plugins/volumebinding/binder_test.go

```diff
@@ -2455,9 +2455,9 @@ func TestGetEligibleNodes(t *testing.T) {
 		eligibleNodes := testEnv.binder.GetEligibleNodes(logger, scenario.pvcs)
 
 		// Validate
-		if reflect.DeepEqual(scenario.eligibleNodes, eligibleNodes) {
```

Comment:
This seems incorrect? Why L2458 has the same content as L2462 ?

Reply:
Sorry, I've mistook.